src/_pytest/terminal.py

```diff
@@ -511,7 +511,9 @@ def pytest_runtest_logstart(
         # Ensure that the path is printed before the
         # 1st test of a module starts running.
         if self.showlongtestinfo:
-            line = self._locationline(nodeid, *location)
```

Comment:
I guess the idea was to pass the entire location, even if one part of it is currently unused?

Reply:
Yeah I wasen't 100% since the `lineno` was never used later, would it be preferred to keep passing `*location` here even if `lineno` is completely unused? happy to revert that too